examples/oauth2/config.json

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
Wouldn't this be a better example if it used the config.json downloaded from the config API, for the user account doing the example. That would make it flexible to whatever the user has configured in 3scale Admin Portal, would avoid having to redact secrets (making sample non-functional), and supply service IDs, shared secrets etc.

The only data data needed would the service_id and token that is already needed by apicast, to get the config from the config API.